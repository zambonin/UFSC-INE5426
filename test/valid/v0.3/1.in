int i = 0, j
float f = 1.1
bool b = true
j = [int] [int] i + f
i = [int] j
b = b & [bool] f
f = ([float] b) + 0.0

