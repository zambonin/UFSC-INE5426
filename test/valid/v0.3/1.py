exec(open('../../../src/scope_manager.py', 'r').read())
i = 0

f = 1.1
b = True
j = int(int((float(i) + f)))
i = int(j)
b = (b & bool(f))
f = (float(b) + 0.0)
