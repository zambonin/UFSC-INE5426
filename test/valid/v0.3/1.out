int var: i = 0, j
float var: f = 1.1
bool var: b = true
= j [int] [int] + [float] i f
= i [int] j
= b & b [bool] f
= f + [float] b 0.0
